src/dotnet/version

```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+0
```

Comment:
Will this conflict with the current already published whisper nugget package?

Reply:
No because it's combined with Whisper's version, which is increasing in this PR.

> // we put our own increments into revision space, so shift the whisper.cpp revision number to make room.
> // (this means we can release 999 versions for every 1 whisper.cpp version.)
> ```
>  let revision = revision * 1_000 + incr
>  $"{major}.{minor}.{revision}"
>  ```

 (Described [here](https://github.com/NickDarvey/whisper/pull/5/files#diff-5fd040457716e3dbd144aad29f0c770ba8f1a070f4002edfb4cbfe516345c223).)